HttpSignature.md

```diff
@@ -1,12 +1,23 @@
-# HTTP-Sig Authentication for SoLiD
+# HTTPSig Authentication for SoLiD
 
-[Signing HTTP Messages](https://tools.ietf.org/html/draft-ietf-httpbis-message-signatures-01)  (henceforth `HTTP-Sig`) is an IETF RFC Draft worked on by the HTTP WG, for signing and authenticating HTTP messages.
+## Executive Summary
+
+HTTPSig is a simple authentication protocol extending [Signing HTTP Messages](https://tools.ietf.org/html/draft-ietf-httpbis-message-signatures-01) (henceforth `HTTP-Sig`) by defining 
```

Comment:
```suggestion
HTTPSig is a simple authentication protocol extending [Signing HTTP Messages](https://tools.ietf.org/html/draft-ietf-httpbis-message-signatures-01) (henceforth `HTTP-Sig`) by defining 
```
`HTTPSig extends HTTP-Sig`?  The extension (here, `HTTPSig`) should get a name that differs more from the thing it's extending than the removal of a hyphen. It's a bit awkward, but `HTTP-Sig-Authn` (or `HTTPSigAuthn` if there's reason not to include hyphens) has enough difference that it's unlikely to appear to be a typo (where `HTTPSig` is reasonably likely to be seen as a typo of `HTTP-Sig`).

Reply:
yes, I agree. I was thinking of calling the "Signing HTTP Messages" document  `HTTP-Sig rfc`, to distinguish it from `HttpSig`  being proposed. HTTP-Sig-Authn would not do, as it is not I think an Authentication mechanism, just a way to sign messages. Perhaps I should just use "Signing HTTP Messages" everywhere.

Reply:
What? The sentence literally states, `HTTPSig` [which I tried to rename `HTTP-Sig-Authn`] "**_is a simple authentication protocol_** extending [Signing HTTP Messages](https://tools.ietf.org/html/draft-ietf-httpbis-message-signatures-01) (henceforth `HTTP-Sig`)..." If `HTTPSig` is not "an Authentication mechanism", then there are much larger changes needed!

Reply:
Ah! sorry. (I just had 6 hours of teleconferencing) Yes, HttpSig is an authentication mechanism. 
I was thinking the other way of keeping HttpSig as the name of the authentication mechanism, and changing the name of the Signing Http Messages from HTTP-Sig to something making the difference greater. Both ways would work.

Reply:
I have replaced all occurrences of HTTP-Sig with [Signing HTTP Messages](https://tools.ietf.org/html/draft-ietf-httpbis-message-signatures-01). I hope that makes things clearer.

Reply:
I'll resolve this one, as I think it is now clear. There may be better ways of doing things, but at least the ambiguity is gone.